### PR TITLE
Adds woof, mrowl, and growl emotes.

### PR DESCRIPTION
**Pull request by tigercat2000**
Alternative to: #3889 

Adds a mrowl emote for Tajara, a woof emote for Vulpkanin, and a growl emote for both of them.

:cl:
rscadd: *growl, *mrowl, and *woof emotes!
/:cl:

### CODE REVIEW

**Comment by LittleBigKid2000** [comment]
inb4 controversy, iamgoofball saying how he doesn't like this, and some mention of snow- aww, too late on that one

**Comment by FlattestGuitar** [comment]
Only woof.ogg is credited, I assume @tigercat2000 recorded the rest himself.

**Comment by tigercat2000** [comment]
@FlattestGuitar The others are licensed under Creative Commons 0.

**Comment by FlattestGuitar** [comment]
I want to dream, tiger!

**Comment by taukausanake** [comment]
I don't... I mean... what?

Tajara mrowl is okaaaay? Sure, okay.

Vulp growl is wat. Not that, needs more growl.

Woof is lame. I'm going to outright say I want something like Minecraft woof. Pretty sure we can't use that though so something similar if possible

**Comment by TheDZD** [comment]
I regret nothing.

**Comment by NTSAM** [comment]
A+ maymay

**Comment by mrturkeytoe** [comment]
https://www.youtube.com/watch?v=BdAUvAXCUW8


End of rine

**Comment by Tastyfish** [comment]
These clips are CC0 because the creator was too embarrassed. :smile:

**Comment by KasparoVy** [comment]
Does this conflict in any way with what might be added in #3889, or is this just an addition on top of that?

**Comment by KasparoVy** [comment]
le lol

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -802,6 +868,8 @@
 						continue
 					M.reagents.add_reagent("space_drugs",rand(1,10))
 
+
+
 		if ("help")
```
Comment (VampyrBytes) [inline code/modules/mob/living/carbon/human/emote.dm]: @tigercat2000 
Can you add them in here please?

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -51,6 +51,21 @@
 			on_CD = handle_emote_CD(50) //longer cooldown
 		if("fart", "farts", "flip", "flips", "snap", "snaps")
 			on_CD = handle_emote_CD()				//proc located in code\modules\mob\emote.dm
+		if("mrowl", "mrowls")
+			if(species.name == "Vulpkanin")
```
Comment (KasparoVy) [inline code/modules/mob/living/carbon/human/emote.dm]: Wrong species :')

**Comment by Fox-McCloud** [comment]
HUE.

**Pull request closed without merging**